src/test/java/com/wikia/webdriver/common/core/elemnt/JavascriptActions.java

```diff
@@ -85,21 +83,22 @@ public void scrollToBottom() {
   }
 
   public void scrollToElement(By elementBy) {
-    try {
-      js.executeScript(
-          "var x = $(arguments[0]);" + "window.scroll(0,parseInt(x.offset().top - 60));",
-          driver.findElement(elementBy));
-    } catch (WebDriverException e) {
-      if (e.getMessage().contains(XSSContent.NO_JQUERY_ERROR)) {
-        PageObjectLogging.log("JSError", "JQuery is not defined", false);
-      }
-    }
+    scrollToElement(driver.findElement(elementBy));
   }
 
   public void scrollToElement(WebElement element) {
+
+    int offset = 60;
+    WikiBasePageObject wikiPage = new WikiBasePageObject();
+    if (wikiPage.isBannerNotificationContainerPresent()){
+      int notificationsHeight = wikiPage.getBannerNotificationsHeight();
+      offset += notificationsHeight;
+    }
+
     try {
       js.executeScript(
-          "var x = $(arguments[0]);" + "window.scroll(0,parseInt(x.offset().top - 60));", element);
+          "var x = $(arguments[0]);" + "window.scroll(0,parseInt(x.offset().top - " + String.valueOf(offset) + "));", element);
+      PageObjectLogging.log("Offset", "Offset is " + String.valueOf(offset), true);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Directly append the argument of String.valueOf(). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1153)